### PR TITLE
feat(GuildMember): add `flags`

### PR DESCRIPTION
**Pull request by almeidx**
**Please describe the changes this PR makes and why it should be merged:**
Upstream:
- discord/discord-api-docs#5874

Depends on:
- discordjs/discord-api-types#700

**Status and versioning classification:**
- I know how to update typings and have done so, or typings don't need updating
- This PR changes the library's interface (methods or parameters added)

### CODE REVIEW

File: packages/discord.js/src/structures/GuildMember.js
Comment (Jiralite) [inline packages/discord.js/src/structures/GuildMember.js:101]: Is this nullable?

**Review by vladfrangu — APPROVED** [review]

File: yarn.lock
Comment (kyranet) [inline yarn.lock:8446]: Small nit, can you dedupe this? `yarn dedupe discord-api-types` should suffice.

Otherwise manually changing this by adding `"discord-api-types@npm:^0.37.27", ` before this YAML key and removing the lines 8439-8444 will also do the trick.

**Review by kyranet — COMMENTED** [review]
Everything else LGTM.

**Review by Jiralite — COMMENTED** [review]
Should this new property be checked in `GuildMember#equals()` too?

**Review by Jiralite — APPROVED** [review]

**Comment by Jiralite** [comment]
Blocking this until #9140 is merged.

**Review by kyranet — APPROVED** [review]

**Review by SpaceEEC — APPROVED** [review]

**Pull request merged**
